script.js

```diff
@@ -1,4 +1,425 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+//Roll Dice Function
+var rollDice = function () {
+  // produces a decimal between 0 and 6
+  var randomDecimal = Math.random() * 6;
+  // take off the decimal
+  var randomInteger = Math.floor(randomDecimal);
+  // it's a number from 0 - 5 ... add 1
+  var diceNumber = randomInteger + 1;
+  return diceNumber;
+};
+// Define Game Modes
+var startGame = `displaying game modes`;
+var selectGameModes = `select game mode`;
+// 2 Player normal
+var twoPlayer_EnterDice = `enter dice roll`;
+var twoPlayer_EnterSeq = `enter sequence of dice`; //not needed anymore
+var twoPlayer_NORMALannounceWinner = `announce winner for normal game before returning to P1`;
+// 2 Player lowest combined num
+var twoPlayerLowest_EnterDice = `enter dice for lowest combined number game`;
+var twoPlayerLowest_EnterSeq = `enter sequence for lowest combined number game`; //not needed anymore
+var twoPlayer_LOWESTannounceWinner = `announce winner for Lowest Combined Number game before returning to P1`;
+var varNumofDice = `mode with variable num of dice`;
+var varNumofDice_rollDice = `roll dice for variable num of dice mode`
+var varNumofDice_announceWinner = `var num of dice - winner`
+// Define initial Game Mode!
+var gameMode = startGame;
+
+// Define global variables
+var diceRolls = [];
+var currPlayer = 1;
+var nextPlayer = 1;
+
+var playerGuess = [];
+var P1Guesses = [];
+var P2Guesses = [];
```

Comment:
I think the namings here are abit off 🤔 . Should probably be `playerDice` in line 33 and so on.